go.mod

```diff
@@ -4,7 +4,6 @@ go 1.18
 
 require (
 	github.com/google/go-cmp v0.5.5
-	github.com/jstemmer/go-junit-report v0.9.1
```

Comment:
This library does not have ability to set `file` attributes. So I switch from the library.